### ISSUE TITLE
SimpleGroup("E6",3); yields strange error message

### ISSUE THREAD

**Issue by Stefan-Kohl** [gap-system/gap#678]
In 4.8.2, I get

```
gap> G := SimpleGroup("E6",2); # o.k.
E6(2)
gap> G := SimpleGroup("E6",3);
Error, Value cannot be set for internal objects in
  SetIsSimpleGroup( g, true ); at /user/GAP/gap4r8/grp/simple.gi:162 called from 
DoAtlasrepGroup( [ s ] ) at /user/GAP/gap4r8/grp/simple.gi:518 called from
<function "SimpleGroup">( <arguments> )
 called from read-eval loop at line 47 of *stdin*
you can 'return;' without setting it
gap> G := SimpleGroup("E6",4); # o.k.
E6(4)
gap> G := SimpleGroup("E6",8);
Error, Value cannot be set for internal objects in
  SetIsSimpleGroup( g, true ); at /user/GAP/gap4r8/grp/simple.gi:162 called from 
DoAtlasrepGroup( [ s ] ) at /user/GAP/gap4r8/grp/simple.gi:518 called from
<function "SimpleGroup">( <arguments> )
 called from read-eval loop at line 48 of *stdin*
you can 'return;' without setting it
```

**Comment by alex-konovalov** [comment]
In this case `g` is `fail` - I think `DoAtlasrepGroup` should check if the call to `AtlasGroup` returns fail and produce a more helpful error message in this case.

**Comment by alex-konovalov** [comment]
@hulpke thanks for the fix. BTW, I can see in the next lines
```
if not '.' in params[1]  then 
  SetIsSimpleGroup( g, true );
fi;
```
Is this assumption safe enough? Is this documented somewhere in AtlasRep or elsewhere?

**Comment by hulpke** [comment]
I see `DoAtlasrepGroup' as a utility function for `SimpleGroup', it will only be called in cases in which this is safe.

**Comment by alex-konovalov** [comment]
This is now part of PR #682 and may be closed when it will be merged.

**Comment by hulpke** [comment]
Fixed in #682
